### PR TITLE
Fold in component library to client

### PR DESCRIPTION
**Pull request by lukenewman**
## Background
Folds in the `@ctoec/component-library` package into the client source. I placed it in `component-library` under `client/src` and am open to discussing relocation.

## GitHub Issue
https://github.com/ctoec/component-library/issues/186

## Associated PRs
N/A

## Validation Plan
Smoke test UI of entire app 💀 

## Tech Specs
- [ ] If any TypeORM entities were created/updated/removed, the necessary migration(s) were [correctly auto-generated](https://github.com/typeorm/typeorm/blob/master/docs/using-cli.md#generate-a-migration-from-existing-table-schema) and **not manually written**.

## Automated Testing
- [ ] All unit tests are passing.
- [ ] Unit tests have been added to cover this changeset.

## Additional Context
N/A

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -6,12 +6,15 @@
     "node": ">=14.18.1 <16.0.0"
   },
   "dependencies": {
-    "@ctoec/component-library": "^2.0.0",
+    "@carbon/icons-react": "^10.44.0",
```
Comment (jessethach) [inline client/package.json:9]: Sweet

**Review by gordonfarrell — APPROVED** [review]
Nice! This is fantastic, burn that component library down! 🔥

**Review by jessethach — APPROVED** [review]

**Comment by jessethach** [comment]
I can validated this PR.

**Comment by jessethach** [comment]
This has been validated with the following smoke test:

As as admin:

- [ ] Navigate to all admin routes successfully
- [ ] Edit user
- [ ] Edit user site permissions. Add org permission.
- [ ] Add/Edit organization
- [ ] Logout

As a provider:

- [ ] Navigate to all non admin routes successfully
- [ ] Upload roster file successfully
- [ ] Edit each child form

- [ ] Non authorized routes work

This PR is ready for merge @nickbristow

**Pull request merged**